sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -40,7 +40,6 @@
   },
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:minify": "uglifyjs -c -m --comments --source-map \"content='./dist/index.js.map'\" -o ./dist/index.min.js ./dist/index.js 2>&1",
```

Comment:
Did these edits to other Key Vault libraries sneak into this PR?

Reply:
Jeremy requested these changes here: https://github.com/Azure/azure-sdk-for-js/pull/9668#discussion_r445151432

And the fix was this: https://github.com/Azure/azure-sdk-for-js/commit/706ff3c9970a711d3805d1969a37a746e5f5ce5e

Since it was so minimal, I thought on doing it here.